### ISSUE TITLE
[AXE-CORE]: ARIA attributes in alerts must conform to valid values

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5961]
## Description

The Parking Alert banner has an `aria-labelledby` attribute that's referencing a missing ID. This is causing an axe-core error and could leave screen reader users confused as to the banner's intent. 

VoiceOver on Safari will not recognize the Parking Alert as a region without this `aria-labelledby` reference being corrected. I suspect NVDA and JAWS will show similar behavior.

Screenshot attached.

## Type of Issue
Template

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a tester, I do not want the aria attribute error to appear on future axe scans

## Environment

* Chrome latest
* MacOS
* https://www.va.gov/pittsburgh-health-care/university-drive-campus-map/
* https://www.va.gov/pittsburgh-health-care/internships-fellowships/forms/
* https://www.va.gov/pittsburgh-health-care/research/safety-security/animal-research/ <br/>This is almost certainly a template update, so I won't include additional items.

## Possible Fixes (optional)

Maybe reference the H3 inside the parking alert banner:

```html
<!-- Change the aria-labelledby to reference the H3 ID -->
<div
  aria-labelledby="parking-alert"
  class="usa-alert-full-width dismissable-option-header usa-alert-full-width-info show-alert"
  id="usa-alert-full-width-2662"
  role="region"
>
  <div class="usa-alert usa-alert-info" role="alert">
    <div class="usa-alert-body">
      <h3 class="usa-alert-heading" id="parking-alert">Parking Alert</h3>
      <div class="additional-info-content usa-alert-text">
        <p>The main entry and exit of our University Drive campus garage will be closed on Monday,
        Feb. 17 while new wayfinding signs are installed. Drivers can enter the garage in the lower
        level entry marked “VA Staff Parking.” Signs will be in place and VA police will be present
        to direct traffic. We apologize for any inconvenience this may cause.</p>
      </div>
    </div>
  </div>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-18 at 3 24 06 PM](https://user-images.githubusercontent.com/934879/74780223-c0054700-5264-11ea-85fb-f04124f0732f.png)

**Comment by ethanteague** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12329

**Comment by mmiddaugh** [comment]
@jenstrickland  and @1Copenut  this one is ready for you

**Comment by jenstrickland** [comment]
I'm not sure how to validate this one without a page that has the Parking Alert notice.

**Comment by mmiddaugh** [comment]
@davidconlon @kevwalsh can we help @jenstrickland test, please?

**Comment by kevwalsh** [comment]
![VA_Pittsburgh_Health_Care___Veterans_Affairs](https://user-images.githubusercontent.com/643678/92643441-c8c87380-f2af-11ea-8f65-e31418bcfd47.jpg)

Here's an updated screenshot using the banner that's live right now.

**Comment by mmiddaugh** [comment]
@jenstrickland We just discussed this situation with the CMS team: 

- The alert capabilities have changed since this particular issue was recorded. 
- When you have a moment, will you please review current alert functionality? (The alert is present across the VAMC pages and is referenced in Kevin's comment above.) We assume the same issue is present but that you may have different proposed solutions given the context and use.

**Comment by jenstrickland** [comment]
@mmiddaugh From @kevwalsh 's screenshot it appears that the necessary id has been added:

`div aria-labelledby="usa-alert-full-width-6922"` is paired with `<h3  id="usa-alert-full-width-6922">`
